src/app/cli/src/init/client.ml

```diff
@@ -1488,31 +1562,87 @@ let export_key =
              Deferred.unit ))
 
 let list_accounts =
-  let open Command.Param in
   Command.async ~summary:"List all owned accounts"
-    (Cli_lib.Background_daemon.graphql_init (return ())
-       ~f:(fun graphql_endpoint () ->
-         let%map response =
-           Graphql_client.query_exn
+    (let%map_open.Command.Let_syntax access_method =
+       choose_one
+         ~if_nothing_chosen:(`Default_to `None)
+         [ Cli_lib.Flag.Uri.Client.rest_graphql_opt
+           |> map ~f:(Option.map ~f:(fun port -> `GraphQL port))
+         ; Cli_lib.Flag.conf_dir
+           |> map ~f:(Option.map ~f:(fun conf_dir -> `Conf_dir conf_dir)) ]
+     in
+     fun () ->
+       let do_graphql graphql_endpoint =
+         match%map
+           Graphql_client.query
              (Graphql_queries.Get_tracked_accounts.make ())
              graphql_endpoint
+         with
+         | Ok response -> (
+           match response#trackedAccounts with
+           | [||] ->
+               printf
+                 "😢 You have no tracked accounts!\n\
+                  You can make a new one using `coda accounts create`\n" ;
```

Comment:
Nit: Should this be `mina accounts create` instead?

Reply:
Good catch, thanks! I've updated to replace this (and a few others) with `mina` :)